data-es6.js

```diff
@@ -1059,6 +1082,28 @@ exports.tests = [
       },
     },
     {
+      name: 'for-loop iteration scope',
```

Comment:
can you add explicit false's to engines that are known to fail, rather than leaving them blank? specifically, v8 iirc fails this